generic/tests/netperf.py

```diff
@@ -290,7 +307,7 @@ def _pin_vm_threads(vm, node):
         src = os.path.join(test.virtdir, params.get("log_hostinfo_script"))
         path = os.path.join(test.resultsdir, "systeminfo")
         process.system_output(
-            "bash %s %s &> %s" % (src, test.resultsdir, path), shell=True)
+            "bash %s %s &> %s" % (src, test.resultsdir, path), shell=True).decode()
```

Comment:
This is meaningless, since you don't use the output, unnecessary to decode it.

Reply:
yes, I tried, it does not needed. I have updated